rosetta-source/src/main/resources/cdm-sample-files/functions/repo-and-bond/repo-substitution.md

```diff
@@ -0,0 +1,10 @@
+### Create Repo Reprice Business Event
```

Comment:
Are the contents of this .md file correct for a substitution? The title looks incorrect but the content looks okay...

Reply:
The md files includes all four events.

Reply:
fixed